docs/next.config.ts

```diff
@@ -58,6 +42,10 @@ try {
 }
 
 export default withDocsInfra({
+  typescript: {
+    // The tsconfig also contains path aliases that are used by next.js.
+    tsconfigPath: IS_PRODUCTION ? '../tsconfig.prod.json' : '../tsconfig.dev.json',
```

Comment:
Couldn't you just override the `paths` in [docs/tsconfig.json](https://github.com/mui/mui-x/blob/master/docs/tsconfig.json) with empty object?

```
   "paths": {}
```

Reply:
Wdym? Not sure what you're suggesting.